README.md

```diff
@@ -275,3 +281,30 @@ crosslinker results in 2 stubs (here for SDA):
      loss:CleavableCrosslinkerPeptide:MASS:0;Name:0
      loss:CleavableCrosslinkerPeptide:MASS:82.04186;NAME:S
 
+#### Changing or adding new entries to the graphical config interface
+The options in the dropdown menu of the interface may be edited according to your needs. 
+The "BasicConfigEntries.conf" contains all the selectable config values. In this file new entries 
+for crosslinker, enzymes, modifications and losses can be freely defined. The file contains sections for crosslinker,
+modifications, losses, ions,enzymes and custom settings. Each section has a short description on how to add new entries.
+
+Additionally there is a file "BasicConfig.conf" containing default values for settings not exposed in the interface.
+
+Both of these are default files that change the options in the dropdown menu of the interface.
```

Comment:
That is not quite true. Settings in BasicConfig.conf do not show up in the interface at all. So any setting there has no effect on the interface (unless you translate the basic-config into the text config) but affect the search as such